src/components/Layout/Foot.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <footer class="footer">
+  <footer id="footer" class="footer">
```

Comment:
Suggestion: A good pattern I've seen used elsewhere is using `data-test-id` attributes instead of `id` proper which reduces the risk of inadvertent side effects (this now creates `window.footer`) and also makes it very clear what the attribute is about, i.e.

```html
<footer data-test-id="footer" class="footer">
```

and then lateron you select the element using `$('[data-test-id="footer"]')`

This comment also applies to all other places where `id`s are introduced.

Reply:
good idea. I also think that it could be troublesome...